scripts/ClearTestResults.ps1

```diff
@@ -21,3 +21,5 @@ RmRF -DirectoryName $testResultsDirName
 foreach ($srcSubdir in (Get-ChildItem "src")) {
     RmRF -DirectoryName "$srcSubdir\$testResultsDirName"
 }
+
+RmRF -DirectoryName $env:Temp -Recurse
```

Comment:
the reason for this one is that many of our tests create temporary files in this directory and don't clean them up after execution. I for one every so often reach a disk limit and at that point tests fail with an exception from the filesystem

Reply:
Maybe this should be a different script? Removing all temp files can be disruptive to other running processes, while removing the test results will only affect the VS and C# dev kit test UIs. This script is currently safe to run at any time, but that would not be the case with this change.

Reply:
I don't understand how would removing temp files be disruptive to other running processes, for the most part if a process has a strong dependency on a file it will open a handle and while the handle is open the delete operation will fail which is an acceptable behavior. I will move this to a separate PR since its not critical for this change request and we can have a discussion there.